scripts/release-deprecate.ts

```diff
@@ -0,0 +1,75 @@
+import { compare } from 'compare-versions';
+import { exec } from 'node:child_process';
+import { readFileSync, readdirSync } from 'node:fs';
+import { join } from 'node:path';
+
+const { log, error } = console;
+
+const UNPUBLISH_TAGS = /next|pr|rc/;
+const { version: CURRENT_VERSION } = JSON.parse(
+  readFileSync(join(process.cwd(), '/packages/fuels/package.json')).toString()
+);
+
+const getPublicPackages = () => {
+  const packagesDir = join(__dirname, '../packages');
+  const packages = readdirSync(packagesDir, { withFileTypes: true });
+  const packagesNames = packages.map((p) => {
+    try {
+      const packageContent = readFileSync(join(packagesDir, p.name, 'package.json'), 'utf8');
+      const packageJson = JSON.parse(packageContent.toString());
+      return packageJson.private ? null : packageJson.name;
+    } catch (err) {
+      return null;
+    }
+  });
+  return packagesNames.filter((p) => !!p);
+};
+
+const main = async () => {
+  const packages = getPublicPackages();
+  await Promise.all(
```

Comment:
```suggestion
  await Promise.allSettled(
```

Should this be all settled?